### DISCUSSION TITLE
Could someone post good settings for a 3090 ?

### DISCUSSION THREAD

**Discussion by metamountain** [kohya-ss/sd-scripts#1549] [General]
have problems in getting it to work... 
dataset size is varying  between 12-40 pictures

Couldnt find a config .json file for a 3090 on the server. Probably quite the same for the 4090 ?

The configuration which worked was very slow and took over 2 hours for 1000 steps...

**Comment by deniaud** [comment]
I'm working on finding the best settings right now and would also be happy to hear recommendations from more experienced users.

**Comment by alcaitiff** [comment]
What are you training? Flux, sd 1.5, sdxl? LoRa, full fine tune?
